src/components/VAlert/VAlert.vue

```diff
@@ -9,14 +9,15 @@
       <!-- @slot The default slot content that is shown to the user -->
       <slot />
 
-      <!-- eslint-disable vue-a11y/click-events-have-key-events -->
       <button
         v-if="dismissible"
         :aria-label="dismissLabel"
         :class="['vts-alert__dismiss', classes.dismiss]"
         @click="dismiss"
+        @keydown.prevent.enter=""
+        @keyup.prevent.enter="dismiss"
```

Comment:
Doesn't this cause the event to fire twice?

Reply:
No, see here: https://github.com/maranran/eslint-plugin-vue-a11y/issues/28#issuecomment-632134646

Reply:
LGTM